cypress/integration/about.spec.js

```diff
@@ -0,0 +1,9 @@
+describe("about", () => {
+  it("should navigate to the about page form the url", () => {
+    // Start from the index page
+    cy.visit("http://localhost:3000/about");
```

Comment:
I think we should configure the base URL `http://localhost:300` in the Cypress config file.